README.md

```diff
@@ -0,0 +1,93 @@
+# tlsdialer
+
+Package go-tlsdialer serves as an interlayer between go-tarantool and go-openssl. 
+All ssl logic from go-tarantool is moved to the go-tlsdialer.
+
+go-tlsdialer still uses tarantool connection, but also types and methods from 
+go-openssl. 
+This way we are removing the direct go-openssl dependency from go-tarantool,
+without creating a tarantool dependency in go-openssl.
```

Comment:
It looks like what we want to achieve with this package, not what this package does. I suspect something like:

```The package allows to create a TLS dialer for `go-tarantool (link)`...```